pkg/kubectl/rolling_updater.go

```diff
@@ -616,11 +615,11 @@ func CreateNewControllerFromCurrentController(rcClient coreclient.ReplicationCon
 	}
 
 	if len(newRc.Spec.Template.Spec.Containers) > 1 && len(cfg.Container) == 0 {
-		return nil, goerrors.New("Must specify container to update when updating a multi-container pod")
+		return nil, fmt.Errorf("Must specify container to update when updating a multi-container pod")
```

Comment:
Can you also lowercase both of these errors? By convention, go errors should start lowercased.